lib/main.dart

```diff
@@ -1,22 +1,29 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_boilerplate/shared/providers/name_notifier.dart';
```

Comment:
@27vinayvyas Use **Relative path** here instead of **absolute path**.

Re-arrange the imports as 
1. Official dart packages
2. Importing the local files in the project